lib/matplotlib/quiver.py

```diff
@@ -85,6 +85,13 @@
     height of the axes, respectively, when the the window is resized;
     for 'dots' or 'inches', resizing does not change the arrows.
 
+  *orientation*: ['absolute' | 'relative']
```

Comment:
The problem with those names is that they imply too much about what the kwarg does; it affects only the orientation, not the whole coordinate system, and not the locations.  Also, the names `axes` and `figure` are themselves ambiguous at best in this context.